Src/EmbeddedScripts.JS.Jint/JintCodeRunner.cs

```diff
@@ -8,36 +8,36 @@ namespace EmbeddedScripts.JS.Jint
     public class JintCodeRunner : ICodeRunner
     {
         public JintCodeRunner(string code)
-            : this(code, _ => JsCodeRunnerOptions.Default)
+            : this(code, _ => CodeRunnerConfig.Default)
         {
         }
 
-        public JintCodeRunner(string code, Func<JsCodeRunnerOptions, JsCodeRunnerOptions> opts)
+        public JintCodeRunner(string code, Func<CodeRunnerConfig, CodeRunnerConfig> configFunc)
         {
             Code = code;
-
-            RunnerOptions = opts(JsCodeRunnerOptions.Default);
+            RunnerConfig = configFunc(CodeRunnerConfig.Default);
         }
 
-        public JintCodeRunner WithOptions(Func<JsCodeRunnerOptions, JsCodeRunnerOptions> opts)
+        public ICodeRunner AddConfig(Func<CodeRunnerConfig, CodeRunnerConfig> configFunc)
```

Comment:
`AddEngineOptions` looks good, but `AddConfig` name confusing. Maybe it will be more user-friendly to expose Register from ICodeRunner:
```
new JintCodeRunner(code)
  .AddEngingeOptions(o => o.strict())
  .Register('x', x)
  .Register('y', y)
  .RunAsync()
```
Eventually it could be transformed to something like that:
```
var engine = ScriptEngineFactory.Get(Engines.JintCodeRunner)
  .AddEngingeOptions(o => o.strict())
  .Register('x', x)
  .Register('y', y)
  .RunAsync(code)

var returnResult = engine.RunAsync("f99()")
```